numba/typed/typedlist.py

```diff
@@ -317,41 +321,41 @@ def __gt__(self, other):
     def __ge__(self, other):
         return _ge(self, other)
 
-    def append(self, item):
+    def append(self, item: T):
```

Comment:
To be fully annotated these should have a `-> None` as well 
https://github.com/python/typeshed/blob/master/stdlib/3/builtins.pyi#L694-L738

Reply:
you're right!